.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow warns and then closes issues and PRs that have had no activity for a specified amount of time.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '21 9 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'There has not been activity on this issue for 60 days. It will be closed automatically in 7 days.'
+        stale-pr-message: 'There has not been activity on this pull request for 60 days. It will be closed automatically in 7 days.'
+        stale-issue-label: 'no-activity'
+        stale-pr-label: 'no-activity'
+        debug-only: true
```

Comment:
Note that this action will run dry runs only until this line is removed.

Reply:
I think we could add `remove-stale-when-updated` to remove the stale label after inactivity

Reply:
I agree, but the default value for `remove-stale-when-updated` is already `true`. I only specified here any changes from defaults values. https://github.com/actions/stale#list-of-input-options